gax/src/main/java/com/google/api/gax/rpc/BidiStreamingCallable.java

```diff
@@ -44,22 +44,189 @@
 
   protected BidiStreamingCallable() {}
 
+  public abstract ClientStream<RequestT> call(
```

Comment:
It appears we are missing a corresponding `call(ResponseObserver<ResponseT> responseObserver, ClientStreamReadyObserver<RequestT> onReady)`

Reply:
I don't think this method is terribly useful for users to call and so maybe it should have the corresponding method you mentioned. Maybe we should document to that effect instead?

Reply:
Ok, we can document it, and reserve the right to add it later.

Reply:
Do we want to call this `internalCall` instead? That might give people more incentive to not use the method.

Reply:
This will need to be called when chaining BidiStreamingCallables, so client developers will need to call it

Reply:
This is very true. Don't mind me :)